arbeitszeit_flask/templates/member/notification-about-work-invitation.html

```diff
@@ -1,7 +1,3 @@
-<p>{{
-    gettext(
-        'You have been invited by a company to join them as a member on Arbeitszeitapp. Accept or reject via this link: "%(invitation_url)s".',
-        invitation_url=invitation_url
-    )
-  }}</p>
+<p>{{ gettext("You have been invited by a company to join them as a member on Arbeitszeitapp. Accept or reject via this link:") }}</p>
+<p><a href="{{ invitation_url }}">{{ invitation_url }}</a></p>
```

Comment:
Good!  But does the link text need to be the bare URL?  Could it be something like "confirm"?  I understand that this is not the goal of this PR.

Reply:
Yeah I think that's a good idea, I'll make an issue for such an enhancement for the emails.